airbyte-integrations/connectors/source-okta/source_okta/manifest.yaml

```diff
@@ -0,0 +1,1882 @@
+            properties:
+              id:
+                type:
+                  - string
+                  - "null"
+            type:
+              - object
+              - "null"
+        additionalProperties: true
+
+  custom_roles_stream:
+    $parameters:
+      name: "custom_roles"
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://{{ config['domain'] }}.okta.com
+        path: /api/v1/iam/roles
+        http_method: GET
+        request_parameters: {}
+        authenticator:
+          $ref: "#/definitions/selective_authenticator"
+        error_handler:
+          $ref: "#/definitions/error_handler"
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - roles
+      paginator:
+        $ref: "#/definitions/paginator"
+      partition_router: []
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        type: object
+        $schema: http://json-schema.org/draft-07/schema#
+        description: Gets a paginated list of Custom Roles
+        properties:
+          id:
+            type:
+              - "null"
+              - string
+          description:
+            type:
+              - "null"
+              - string
+          label:
+            type:
+              - "null"
+              - string
+          created:
+            format: date-time
+            type:
+              - "null"
+              - string
+          lastUpdated:
+            format: date-time
+            type:
+              - "null"
+              - string
+          _links:
+            type:
+              - "null"
+              - object
+            properties:
+              assignee:
+                type:
+                  - "null"
+                  - object
+                additionalProperties: true
+                properties:
+                  self:
+                    type:
+                      - "null"
+                      - object
+                    additionalProperties: true
+                    properties:
+                      href:
+                        type:
+                          - "null"
+                          - string
+                  permissions:
+                    type:
+                      - "null"
+                      - object
+                    additionalProperties: true
+                    properties:
+                      href:
+                        type:
+                          - "null"
+                          - string
+        additionalProperties: true
+
+check:
+  type: CheckStream
+  stream_names:
+    - users
+
+streams:
+  - type: DeclarativeStream
+    name: groups
+    $ref: "#/definitions/groups_stream"
+
+  - type: DeclarativeStream
+    name: group_members
+    $parameters:
+      name: "group_members"
+    primary_key:
+      - type
```

Comment:
Why have you changed the primary key? This is a breaking change

Reply:
Fixed this, the primary key is back to the one used in the Python implementation